tardis/montecarlo/montecarlo_numba/montecarlo_logger.py

```diff
@@ -22,6 +22,50 @@
 logger.addHandler(console_handler)
 
 
+def logging_state(log_state):
+    """
+    The configuration for the Logging status of the run() simulation object
+    Called from run_tardis()
+    Invoked via Function, argument 
+    Proposed : Invoking via YAML & Flags
+
+    Parameters
+    ----------
+    log_state: boolean or string
```

Comment:
I am not in favour of parameters that have multiple types. It is unclear for the user even with documentation. Instead set a default string e.g. `log_state = "CRITICAL"`

Reply:
Sure, easier way to keep track of the logging 🚀